test/unit/src/geometries/CircleGeometry.tests.js

```diff
@@ -59,16 +59,39 @@ export default QUnit.module( 'Geometries', () => {
 
 		} );
 
-		QUnit.todo( 'parameters', ( assert ) => {
+		// PARAMETERS
+		QUnit.test( 'parameters', ( assert ) => {
 
-			assert.ok( false, 'everything\'s gonna be alright' );
+			assert.ok(
+				geometries[ 1 ].parameters.radius === 10,
+				geometries[ 2 ].parameters.radius === 10,
+				geometries[ 2 ].parameters.segments === 20,
+				geometries[ 3 ].parameters.radius === 10,
+				geometries[ 3 ].parameters.segments === 20,
+				geometries[ 3 ].parameters.thetaStart === 0.1,
+				geometries[ 4 ].parameters.radius === 10,
+				geometries[ 4 ].parameters.segments === 20,
+				geometries[ 4 ].parameters.thetaStart === 0.1,
+				geometries[ 4 ].parameters.thetaLength === 0.2
+			);
 
 		} );
 
 		// STATIC
-		QUnit.todo( 'fromJSON', ( assert ) => {
+		QUnit.test( 'fromJSON', ( assert ) => {
+
+			const geometriesJSON = '{"parameters": {"radius": "0", "segment": "0", "thetaStart": "0", "thetaLength": "0"}}';
```

Comment:
What is your motivation for using `0` values everywhere? The resulting geometry will be empty and thus useless: https://jsfiddle.net/fa048vnk/